pyface/timer/tests/test_timer.py

```diff
@@ -199,7 +199,9 @@ def tearDown(self):
 
     def test_basic(self):
         handler = ConditionHandler()
-        timer = CallbackTimer(callback=handler.callback)
+        timer = CallbackTimer(
+            callback=handler.callback, kwargs={'event': None}
+        )
```

Comment:
it is better to simply update the `ConditionHandler.callback` method to set `event=None` instead of updating all of the callers

Reply:
That makes sense to me, should I do it where I simply have `event=None` as a kwarg on `callback` then?  Or is the most recent solution with `_callback` better?

I think just having it as a kwarg makes the most sense to me

Reply:
> I think just having it as a kwarg makes the most sense to me

Let's go with that approach - instead of the refactor.